File: AsalTask/ViewControllers/Photos/FullPhotoViewController.m
//
//  FullPhotoViewController.m
//  AsalTask
//
//  Created by Amr Abu Zant on 10/14/18.
//  Copyright © 2018 Amr Abu Zant. All rights reserved.
//

#import "FullPhotoViewController.h"

@interface FullPhotoViewController ()

@end

@implementation FullPhotoViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self loadPhoto:_photo];
}

- (void) loadPhoto:(Photo*) photo{
    [_photoImageView sd_setImageWithURL:[NSURL URLWithString:photo.url]];
    [self setTitle:photo.title];
}

@end

File: AsalTask/ViewControllers/Photos/FullPhotoViewController.swift
//
//  FullPhotoViewController.swift
//  AsalTask
//
//  Created by Amr Abu Zant on 10/13/18.
//  Copyright © 2018 Amr Abu Zant. All rights reserved.
//

import UIKit
import SDWebImage

class FullPhotoViewController: UIViewController {

    @IBOutlet weak var photoImageView: UIImageView!
    
    var photo : Photo? = nil
    
    override func viewDidLoad() {
        super.viewDidLoad()

        if let data = photo {
            loadPhoto(photo: data)
        }
    }
    
    func loadPhoto(photo : Photo){
        photoImageView.sd_setImage(with: URL(string: photo.url))
        self.title = photo.title
    }

}
